docs/quickstart.ipynb

```diff
@@ -4,20 +4,18 @@
    "cell_type": "markdown",
```

Comment:
<p>I think the information about executing the notebook on a server is worth keeping. It's useful for users new to python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I found this instruction odd, I always think about a user with a new Jupyter Notebook (or Python script) reproducing the commands listed here, not running this document specifically.</p>